docs/development/modules.md

```diff
@@ -356,6 +356,24 @@ druid.coordinator.cleanupMetadata.duty.killSupervisors.retainDuration=PT0M
 druid.coordinator.cleanupMetadata.period=PT10S
 ```
 
+### Routing data through a HTTP proxy for your extension
+
+It is possible to adds the ability for the HttpClient of your extension to connect through a HTTP proxy. 
+Internally, Druid will augment the channel factory to check if it is supposed to be proxied and, if so, connect to the proxy host first, 
+issue a CONNECT command through to the final recipient host and then give the channel to the normal http client for usage.
```

Comment:
```suggestion
You can add the ability for the `HttpClient` of your extension to connect through an HTTP proxy. 
Internally, Druid augments the channel factory to check if it should proxy connections. If so the client connects to the proxy host first to issue a subsequent `CONNECT` command through to the final recipient host. After the connection is established, the channel passes back to the original HTTP client for usage.
```
I'm not sure what "normal" means in the connection here. It seems like the way that Druid handles it by passing the channel back and forth may be an implementation detail. I am wondering if I am enabling proxying why I need to know that?

Reply:
Reworded as suggestion and removed implementation details